src/structures/search.rs

```diff
@@ -0,0 +1,68 @@
+use super::*;
+use crate::structures::project::{ProjectSupportRange, ProjectType};
+
+#[derive(Deserialize, Serialize, Debug, Clone)]
+pub struct SearchResult {
```

Comment:
This looks identical to `structures::project::Project`, so you can just use that.

Reply:
There are some differences between that and project, `id` is `project_id`, `follows` is `followers` and (maybe?) `display_categories` is `additional_categories`.

Theres also some fields that are missing from `Project`: `source_url`, `wiki_url`, `discord_url`, `donation_urls`, `body_url`, `team`.

At a first glance it really looks identical to `structures::project::Project`, and while i was implementing it was really a thing that crossed my mind, but no the structures are different.